### DISCUSSION TITLE
关于抵押

### DISCUSSION THREAD

**Discussion by pzjboy0924** [tearust/Chinese-wiki#7] [General]
根据wiki上的说明，抵押者的收益weight永远不能超过矿工的weight，无论多少人抵押，抵押多少次。这里就产生了一个问题，如果TAPP所有者用自己的种子B去host自己的TAPP，挖矿产生的收益永远比抵押者的收益高，他就可以用这些收益继续购买自己的token来提高TAPP的总市值，其他人没有办法降低这个TAPP的总市值，除了他自己。只要他一直买进自己的token，那就意味着永远保持第一名。

出现上面的情况，是不是可以用比他自己多的B去host他的TAPP，争取获得更多的挖矿任务，从而获取到更多的token，收益有没有可能就超过开发者?

然后会出现另外一种情况开发者自己用B种子把自己的TAPP垄断，这需要开发者有绝对的自信。

**Comment by kevingzhang** [comment]
其他人没法降低你的tapp的市值, 但是别的Tapp会增加市值啊. 增加市值最重要的是来自消费者的盈利而不是投资. 一个没有盈利只有自己给自己投资的TApp你自己也挣不到钱啊. 如果被人都不买你的应用, 你投入的钱还是你自己的, 那么意义何在呢? 

我估计你没有明白这些token最终是怎么来的. 这些token最终是来自于消费者. 消费者使用TApp付费. 这些利润进入TApp的bondingcurve, 推高所有人的token总数和币价. 而不是来自于开发者自己掏钱投资给自己, 然后自己掏钱消费 把自己消费的利润从左面口袋搬到右边口袋再次投资进去. 这样还是你自己的钱, 没有增值.

**Reply by pzjboy0924** [reply]
目前这个竞赛模式就是这样的规则，正式上线后，为了抢夺排名也许还会短暂出现这种情况

**Comment by kevingzhang** [comment]
这种情况没啥问题啊. 我可能没有理解你提出这个问题的原因. 一个人愿意自己掏钱把自己的应用捧起来没啥错误. 就好比一个人掏钱组织一个演唱会请人来听自己唱歌这个不违反任何法律啊. 你的TApp推到排名第一不会给你自己增加收益. 如果是我, 我不会这么做的. 我有这些钱一定会投资给那些有利润收入可以带来现金流的应用, 而不是我自己的肯定没有人花钱的应用.

**Reply by pzjboy0924** [reply]
推到第一有种子奖励呀！

**Comment by kevingzhang** [comment]
奖励不是你的应用到第一, 而是你的财产排名第一. 你看leaderboard的排名. 而不是看tapp的排名

**Reply by pzjboy0924** [reply]
我一直说的是TAPP的总市值，没说tapp排名。

**Reply by kevingzhang** [reply]
TApp 的总市值和你的资产排名不是一回事儿.
我可以一个tapp也不建立, 一个app也不投资. 只要我钱多, 一样可以排在前面.
挣钱的方式很多. app是其中一种, 虽然我认为是最重要的, 最推荐的一种.
所以leaderboard的排名不是看你的app的排名, 是你的资产的排名.
